src/main/resources/com/google/jenkins/plugins/computeengine/ComputeEngineComputer/configure.jelly

```diff
@@ -25,7 +25,7 @@
                 <j:set var="APIInstance" value="${it.instance}" />
                 <j:set var="cloud" value="${instance.cloud}" />
 
-                <h3>This is a read-only view. Click <a href="${rootURL}/configure"> here</a> to configure your instances.</h3>
+                <h3>This is a read-only view. Click <a href="${rootURL}/configureClouds/"> here</a> to configure your instances.</h3>
```

Comment:
When testing this locally with mvn hpi:run, this didn't work. This is meant to point at the configure system page, which should be at "/configure"

Reply:
@daniel-beck changed this as of https://github.com/jenkinsci/jenkins/commits/jenkins-2.205 by https://github.com/jenkinsci/jenkins/pull/4339 especially https://github.com/jenkinsci/jenkins/commit/77f1242ddd2dd4307b788e86422ace8acd1c8445

It looks like LTS is currently 2.204.1 which means it just missed.

I guess once the next LTS arrives, I can include a change to the pom to bump the jenkins version which should result in your testing working.
https://github.com/jenkinsci/google-compute-engine-plugin/blob/19e6cc806f21ab23ea1ceddd68dbb4c2e589db8a/pom.xml#L70

Alternatively, I suppose w/ some amount of magic, I could ask jenkins if an endpoint exists and conditionally return the appropriate url.

Reply:
FWIW there's a placeholder in the existing configure page, so it's not like this is now _wrong_, just adds a level of indirection.

Reply:
From a UX perspective, getting a link to a page which just says _you shouldn't be here_ is fairly insulting. That said, if the code path is inconsistent at this point, then either a branch in the code or a version bump at the next LTS seems warranted.

Reply:
jenkins version is currently set to [`2.401.3`](https://github.com/jenkinsci/google-compute-engine-plugin/blob/fa446a7e00ad7ef0e0e577f65deef677910b5299/pom.xml#L68)  so this has long since past :)

Reply:
This URL has also been subsequently changed https://github.com/jenkinsci/jenkins/pull/7658/ in Jenkins version [jenkins-2.403](https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.403) to be `manage/cloud/` 

Allthough there is a HTTP redirect in place for this.

Reply:
https://github.com/jenkinsci/google-compute-engine-plugin/pull/457